modules/rhcos-environment-variables.adoc

```diff
@@ -29,11 +29,14 @@ You can use the following environment variables to configure {op-system} node sc
 |====
 |Environment Variable|Description
 
+|ROX_NODE_INDEX_ENABLED
+|Controls whether node indexing is enabled for this cluster. The default value is `false`. This variable is used for Scanner V4.
+
 |ROX_NODE_SCANNING_INTERVAL
-|The base value of the interval duration between node scans. The deafult value is `4h`.
+|The base value of the interval duration between node scans. The default value is `4h`.
 
 |ROX_NODE_SCANNING_INTERVAL_DEVIATION
-|The duration of node scans may differ from the base interval time. However, the maximum value is limited by the `ROX_NODE_SCANNING_INTERVAL`.
+|The duration of node scans might differ from the base interval time. However, the maximum value is limited by the `ROX_NODE_SCANNING_INTERVAL`.
```

Comment:
@Maddosaurus does this set some jitter value? if so, then I think the original "may" is better than "might", as it's more definitive (this *will*happen). May even be better to just explicitly say this sets the bounds for the amount of jitter we add